### DISCUSSION TITLE
Very large vertical spacing in the Drawer

### DISCUSSION THREAD

**Discussion by HubKing** [LawnchairLauncher/lawnchair#4886] [Q&A] [answered]
I updated to the "dev" version on the Play Store, and noticed that the vertical spacing is very large. It almost looks like the height of the icon, and I could not find any setting to reduce it. Is this a bug or a feature?

![Screenshot_20241012-045631_Lawnchair](https://github.com/user-attachments/assets/41e8fce1-98e0-4ad2-ada4-4793ba3e81a5)

**Comment by RastalDev** [comment, marked as answer]
Settings > App drawer > Grid > Row Height.

**Reply by HubKing** [reply]
That setting seems to have no effect. I set it to 30% (the minimum), and the vertical spacing was the same.

**Reply by RastalDev** [reply]
Should work. If it doesn't work on your device then you've discovered a bug, best bet is to submit an Issue for it.

**Reply by SuperDragonXD** [reply]
Hello. As this discussion has been inactive for a while, I'm going to close this. If you are still experiencing this issue in Lawnchair 15, please create a new discussion or issue.
